pkg/cmd/util/util.go

```diff
@@ -295,37 +297,37 @@ func (d *disabledGrpcServer) NetDialContext(_ context.Context, _ string) (net.Co
 func (d *disabledGrpcServer) GracefulStop() {}
 
 type HTTPServerConfig struct {
-	Address     string
-	TLSCertPath string
-	TLSKeyPath  string
-	Enabled     bool
+	HTTPAddress     string `debugmap:"visible"`
+	HTTPTLSCertPath string `debugmap:"visible"`
+	HTTPTLSKeyPath  string `debugmap:"visible"`
+	HTTPEnabled     bool   `debugmap:"visible"`
```

Comment:
why was the prefix needed?

Reply:
Because over method name overlap since we're generating this and the grpc struct in the same module

Reply:
It that's the case I think we should probably add `GRPC` prefix to `GRPCServerConfig` fields to keep everything homogeneous?